core/src/data/value/binary_op/f64.rs

```diff
@@ -1,16 +1,32 @@
 use {
     super::TryBinaryOperator,
     crate::{data::ValueError, prelude::Value, result::Result},
+    rust_decimal::prelude::Decimal as Dec,
     std::cmp::Ordering,
     Value::*,
 };
 
+impl PartialEq<Value> for f64 {
```

Comment:
There was something we were missing! thanks for the good catch

Reply:
+1  I wondered why it was in i64, and realized it must have been a mistake.. ;)